README.md

```diff
@@ -1,25 +1,68 @@
+    <p align="center">
+        <a href="https://travis-ci.org/zaquestion/lab">
+            <img src="https://travis-ci.org/zaquestion/lab.svg?branch=master" alt="Build Status">
+        </a>
+        <a href="https://goreportcard.com/report/github.com/zaquestion/lab">
+            <img src="https://goreportcard.com/badge/github.com/zaquestion/lab" alt="Go Report Card">
+        </a>
+        <a href="https://codecov.io/gh/zaquestion/lab">
+            <img src="https://codecov.io/gh/zaquestion/lab/branch/master/graph/badge.svg" alt="codecov">
+        </a>
+        <a href="https://gitter.im/labcli">
+            <img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join the chat">
+        </a>
+    </p>
+    <p align="center">
+        <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">
+            <img src="http://i.creativecommons.org/p/zero/1.0/88x31.png" alt="CC0 License">
+        </a>
+        <a href="https://liberapay.com/zaquestion/donate">
+            <img src="https://liberapay.com/assets/widgets/donate.svg" alt="Donate">
+        </a>
+    </p>
+    <p align="center">
+        <img src="https://user-images.githubusercontent.com/1964720/42740177-6478d834-8858-11e8-9667-97f193ecb404.gif" align="center">
+    </p>
+</p>
 
-<p align="center"><img src="https://user-images.githubusercontent.com/1964720/42740177-6478d834-8858-11e8-9667-97f193ecb404.gif" align="center"></p>
+_lab_ interact with repositories on GitLab, including creating/editing merge requests, issues, milestones, snippets and
+CI pipelines.
 
-Lab wraps Git, making it simple to clone, fork, and interact with repositories on GitLab, including seamless workflows for creating merge requests, issues and snippets.
+The development team has the focus on keeping _lab_ a simple, the less opinionated possible, command line interface for
+what the GitLab API offers. Also, _lab_ aims to feel familiar to a Git user and follow Git's behaviors/patterns for
+some of its commands and flags.
 
-```
-$ lab clone gitlab-com/infrastructure
+# Usage recommendation
 
-# expands to:
-$ git clone git@gitlab.com:gitlab-com/infrastructure
+One can use _lab_ as a Git alias, integrating seamlessly to their Git workflow.
+```
+$ cat ~/.gitconfig
+...
+[alias]
+    lab = "!lab"
+    lab-i = "!lab issue"
+    li = "!lab issue"
+
+$ git lab mr list
+$ git lab-i close
+$ git li create
 ```
 
-# Inspiration
+Also, _lab_ can be set as shell aliases:
 
-The [hub](https://github.com/github/hub) tool made my life significantly easier and still does! lab is heavily inspired by hub and attempts to provide a similar feel.
+```bash
+alias mrlist="lab mr list"
+```
 
 # Installation
 
-Dependencies
-
-* `git`
+In compilation-time, _lab_ depends only on other Go external modules, defined at `go.mod`. But at runtime some Git
+commands are used underneath for retrieving local repository information. Because of that, make sure you have _git_
+installed in your system for running _lab_.
 
```

Comment:
In compilation-time, _lab_ depends only on other Go external modules, defined at `go.mod`.  At runtime, _git_ is required as many _git_ commands are used by lab to retrieve local repository information.

Reply:
Thanks!